pkg/config/app_config.go

```diff
@@ -311,33 +311,6 @@ func loadAppState() (*AppState, error) {
 	return appState, nil
 }
 
-// AppState stores data between runs of the app like when the last update check
-// was performed and which other repos have been checked out
-type AppState struct {
```

Comment:
I don't see a need to move this to its own file; I reverted this in my fixup, just because it makes the change smaller.